static/js/stream_edit.js

```diff
@@ -81,9 +81,16 @@ function get_sub_for_target(target) {
     return sub;
 }
 
+function no_users_to_add(sub) {
```

Comment:
This would be much better named `update_no_users_display`.  But I'm also not sure this is a great way to do it; couldn't we use the "message on empty" feature of the `list_render` widget instead?